torch/csrc/autograd/python_function.cpp

```diff
@@ -44,19 +44,24 @@ PyObject *THPFunctionClass = nullptr;
 
```

Comment:
Should we fix this too?

Reply:
we don't need to, for a similar reason https://github.com/pytorch/pytorch/pull/34845/files#r393387465